portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -78,6 +240,52 @@ const loadMoreProjects =  async () => {
     projects.scrollTo({ top: projects.scrollHeight, behavior: "smooth" });
 };
 
+const loadMap = () => {
```

Comment:
nit: the name of this file is `queries/index.js`, but this function is not related to queries. Consider moving the map functionality to a different file.

Reply:
What would be a nice approach?

It's cause, normally, I'd use the "queries" folder as a suite of functions to fetch any type of info (a server endpoint or an API endpoint). But I would agree that technically they are not the same.

I could probably create a folder for this API requests that don't go to the webapp's server. WDYT?

Reply:
I would recommend splitting out the files by which different services they interact with. In that case, it would make sense to move the functions related to maps to a separate directory named something like `maps/index.js`.